src/LegacyConfig.php

```diff
@@ -36,6 +49,10 @@ public function __construct($config)
         }
     }
 
+    /**
+     * @param mixed $parameters
+     * @return mixed
```

Comment:
This should be an array of `mixed`

```suggestion
     * @return mixed[]
```